addons/survey/views/survey_templates.xml

```diff
@@ -421,139 +419,158 @@
+                </div>
+                <!--Go to line for vertical list of answers -->
+                <div t-if="use_single_column or (print_mode and not use_half_columns)" class="w-100 m-0"/>
+            </t>
+            <t t-if='question.comments_allowed and question.comment_count_as_answer'>
+                <div t-att-class="'col-sm-12 %s' % ('col-lg-6' if use_half_columns or print_mode else '')">
+                    <label t-att-class="'o_survey_choice_btn py-1 px-3 h-100 w-100 rounded %s' % ('o_survey_selected' if comment_line else '')">
                         <t t-set="item_idx" t-value="item_idx + 1"/>
                         <t t-call="survey.survey_selection_key">
                             <t t-set="selection_key_class" t-value="'position-relative float-start d-flex'"/>
                         </t>
+                        <i class="fa fa-check-circle float-end mt-1 ms-1 position-relative"></i>
+                        <i class="fa fa-circle-thin float-end mt-1 ms-1 position-relative"></i>
                         <input type="checkbox" class="o_survey_form_choice_item o_survey_js_form_other_comment invisible position-absolute" value="-1"
-                               t-att-name="question.id"
-                               t-att-checked="comment_line and 'checked' or None"
-                               t-att-data-selection-key="letters[item_idx] if useKeySelection else ''"/>
+                            t-att-name="question.id"
+                            t-att-checked="comment_line and 'checked' or None"
+                            t-att-data-selection-key="letters[item_idx] if useKeySelection else ''"/>
                         <span class="ms-2" t-out="question.comments_message or default_comments_message" />
-                        <i class="fa fa-check-circle float-end mt-1 position-relative"></i>
-                        <i class="fa fa-circle-thin float-end mt-1 position-relative"></i>
                     </label>
                 </div>
-                <div t-attf-class="o_survey_comment_container mt-3 py-0 px-1 #{'d-none' if not comment_line else ''}">
+                <!--Go to line for vertical list of answers -->
+                <div t-if="use_single_column or (print_mode and not use_half_columns)" class="w-100 m-0"/>
+                <div t-attf-class="o_survey_comment_container mt-3 py-0 px-1  #{'d-none' if not comment_line else ''}">
                     <textarea type="text" class="form-control o_survey_question_text_box bg-transparent text-dark rounded-0 p-0"
-                              t-att-disabled="None if comment_line else 'disabled'"><t t-esc="comment_line.value_char_box if comment_line else ''"/></textarea>
+                            t-att-disabled="None if comment_line else 'disabled'"><t t-esc="comment_line.value_char_box if comment_line else ''"/></textarea>
                 </div>
```

Comment:
At the moment, with your changes, the textarea is always full-width large. But I think it may be ok to give more room to write down comments, so I let the future testers give their opinion on this one :)